src/shared/lib/onMobxPromise.ts

```diff
@@ -1,16 +1,6 @@
 import {MobxPromise} from "mobxpromise";
 import {autorun, IReactionDisposer} from "mobx";
 
-export function onMobxPromise<T>(promise:MobxPromise<T>,
```

Comment:
This is intensional?

Reply:
yes, theres actually no need for multiple signatures, and its causing problems so I just got rid of them